analyses/focal-cn-file-preparation/02-add-ploidy-consensus.Rmd

```diff
@@ -45,7 +50,11 @@ if(!dir.exists(output_dir)) {
 # gets included we will have to change the file path here
 consensus_seg_file <- file.path("..", "copy_number_consensus_call", "results", 
                                 "pbta-cnv-consensus.seg.gz")
+if ( params$base_run ==0 ){
 histologies_file <- file.path("..", "..", "data", "pbta-histologies.tsv")
+} else {
+histologies_file <- file.path("..", "..", "data", "pbta-histologies-base.tsv")
+}
```

Comment:
Per https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1130/files#r686770033:

```suggestion
if ( params$base_run ==0 ){
  histologies_file <- file.path("..", "..", "data", "pbta-histologies.tsv")
  consensus_seg_file <- file.path("..", "..", "data", 
                                  "pbta-cnv-consensus.seg.gz")
} else {
  histologies_file <- file.path("..", "..", "data", "pbta-histologies-base.tsv")
  consensus_seg_file <- file.path("..", "copy_number_consensus_call", "results", 
                                  "pbta-cnv-consensus.seg.gz")
}
```

 I can not include removing the TODO comment in my suggestion for this diff, so let's make sure we do that too!